app/views/layouts/govuk_publishing_components/application.html.erb

```diff
@@ -13,11 +13,6 @@
   <%= stylesheet_link_tag "component_guide/application", media: "screen" %>
   <%= stylesheet_link_tag "#{GovukPublishingComponents::Config.application_stylesheet}" %>
 
-  <% if GovukPublishingComponents::Config.static %>
-    <%= stylesheet_link_tag "component_guide/all_components", media: "screen" %>
-    <%= stylesheet_link_tag "component_guide/all_components_print", media: "print" %>
```

Comment:
If we're removing these references we should remove them from the component gem as well.